scryfall_scraper.py

```diff
@@ -16,30 +16,56 @@ def get_scryfall(setUrl):
             scryfall.append(setcards['data'])
         else:
             setDone = True
-            # print setUrl
-            # print setcards
```

Comment:
Printing the api url where we get the data from doesn't look too bad :)

`print setcards` would print all card names though?!

Reply:
If everything's working, it's excessive debug printing